pkg/component/runtime/manager_test.go

```diff
@@ -1564,7 +1588,8 @@ LOOP:
 		case <-endTimer.C:
 			// no deadlock after timeout (all good stop the component)
 			updatedCancel()
-			_ = m.Update(component.Model{Components: []component.Component{}})
+			m.Update(component.Model{Components: []component.Component{}})
+			<-errCh // Don't care about the result of Update, just that it runs
```

Comment:
This is where we have this test hanging in CI... 
Shouldn't this be 
```suggestion
			m.Update(component.Model{Components: []component.Component{}})
			<-m.errCh // Don't care about the result of Update, just that it runs
```
since `errCh` is a channel used internally in the test ?